clients/go/coreutils/literals.go

```diff
@@ -521,6 +539,11 @@ func MakeLiteralForType(t *core.LiteralType, v interface{}) (*core.Literal, erro
 		lv := MakeLiteralForBlob(storage.DataReference(fmt.Sprintf("%v", v)), isDir, newT.Blob.Format)
 		return lv, nil
 
+	case *core.LiteralType_Schema:
+		newT := t.Type.(*core.LiteralType_Schema)
```

Comment:
If you change this line:
```golang
	switch t.Type.(type) {
```
to:
```golang
	switch newT := t.Type.(type) {
```

You can delete this line:
```suggestion
```

And the ones like it in the other `case` blocks up there...

Reply:
Done